.github/workflows/ci.yml

```diff
@@ -32,16 +57,14 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r test-requirements.txt
 
-      - name: Run tests
+      - name: Test typing
+        if: matrix.python-version < '3.7'
```

Comment:
typing has been in the stdlib since 3.5, so this should only be run on 2.7 and (for now) 3.4

Reply:
Considering that we don't run CI for 3.4 at the moment (since it doesn't seem to exist), we can just remove this check completely (except for Python 2.7).